css-two-column-layout/styles.css

```diff
@@ -0,0 +1,89 @@
+body {
+  font-family: Karma, sans-serif;
+  letter-spacing: 2px;
+}
+
+.container {
+  margin: 5%;
+}
+
+.center-elem {
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 0 15%;
+}
+
+.header-section {
+  margin-bottom: 20px;
+}
+
+.row {
+  display: flex;
+}
+
+.col-half {
+  flex-basis: 45%;
+}
+
+.img-col {
+  flex-basis: auto;
+  margin-right: 5px;
+}
```

Comment:
Why not just make this a `col-third` so that you are using a proper layout class system?